certified-connectors/Business Assist/apiDefinition.swagger.json

```diff
@@ -0,0 +1,688 @@
+                  "type": "integer",
+                  "format": "int32",
+                  "description": "Clustering Status"
+                },
+                "Diagnostics": {
+                  "type": "string",
+                  "description": "Diagnostics message"
+                }
+              }
+            }
+          }
+        },
+        "summary": "Get text analytic job status",
+        "description": "Get text analytic job status by job id.",
+        "operationId": "GetClusteringStatus",
+        "x-ms-visibility": "internal",
+        "parameters": [
+          {
+            "name": "Clustering job Id",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "a6fe7af6-ba42-4676-9a4c-901d0bd44d19",
+            "x-ms-url-encoding": "single"
+          }
+        ]
+      }
+    },
+    "/api/beta/textanalytics/topicclustering/{Clustering job Id}/result": {
+      "get": {
+        "responses": {
+          "200": {
+            "description": "default",
+            "schema": {
+              "type": "object",
+              "properties": {
+                "OperationId": {
+                  "type": "string",
+                  "description": "OperationId"
+                },
+                "Status": {
+                  "type": "integer",
+                  "format": "int32",
+                  "description": "Status"
+                },
+                "Results": {
+                  "type": "array",
+                  "items": {
+                    "type": "object",
+                    "properties": {
+                      "RecordId": {
+                        "type": "string",
+                        "description": "RecordId"
+                      },
+                      "Cluster_Id": {
+                        "type": "integer",
+                        "format": "int32",
+                        "description": "Cluster_Id"
+                      },
+                      "Cluster_Probability": {
+                        "type": "number",
+                        "format": "float",
+                        "description": "Cluster_Probability"
+                      },
+                      "Cluster_Label": {
+                        "type": "string",
+                        "description": "Cluster_Label"
+                      }
+                    }
+                  },
+                  "description": "Results"
+                }
+              }
+            }
+          }
+        },
+        "summary": "Get text analytic job result",
+        "description": "Get text analytic job results by using job id.",
+        "operationId": "GetClusteringResult",
+        "x-ms-visibility": "internal",
+        "parameters": [
+          {
+            "name": "Clustering job Id",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "a6fe7af6-ba42-4676-9a4c-901d0bd44d19",
```

Comment:
@sjagaleti please change the value of the description. Secret look alike values are not allowed.